### PR TITLE
Fix DateTimePicker example and translation

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix https://github.com/nextcloud/nextcloud-vue/issues/3403
Preview: https://deploy-preview-3405--nextcloud-vue-components.netlify.app/#/Components/NcPickers?id=ncdatetimepicker

Regression of https://github.com/nextcloud/nextcloud-vue/pull/3202

### CODE REVIEW

File: src/components/NcDatetimePicker/NcDatetimePicker.vue [hunk 1]
```diff
@@ -280,6 +280,8 @@ export default {
 		return {
 			showTimezonePopover: false,
 			tzVal: this.timezoneId,
+
```
Comment (szaimen) [inline src/components/NcDatetimePicker/NcDatetimePicker.vue:283]:
```suggestion
```

**Review by szaimen — APPROVED** [review]

**Review by szaimen — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

CI fails though

**Review by Pytal — APPROVED** [review]

**Comment by skjnldsv** [comment]
> CI fails though

Cypress failures unrelated

**Comment by PVince81** [comment]
> > CI fails though
> 
> Cypress failures unrelated

raised #3430

**Pull request merged**